src/rpm/builder.rs

```diff
@@ -264,17 +264,17 @@ impl PackageBuilder {
     /// let pkg = rpm::PackageBuilder::new("foo", "1.0.0", "Apache-2.0", "x86_64", "some baz package")
     ///     .with_file(
     ///         "./awesome-config.toml",
-    ///         rpm::FileOptions::new("/etc/awesome/config.toml").is_config(),
+    ///         rpm::FileOptions::regular("/etc/awesome/config.toml").is_config(),
     ///     )?
     ///     // file mode is inherited from source file
     ///     .with_file(
     ///         "./awesome-bin",
-    ///         rpm::FileOptions::new("/usr/bin/awesome"),
+    ///         rpm::FileOptions::regular("/usr/bin/awesome"),
     ///     )?
     ///      .with_file(
     ///         "./awesome-config.toml",
     ///         // you can set a custom mode, capabilities and custom user too
-    ///         rpm::FileOptions::new("/etc/awesome/second.toml").mode(0o100744).caps("cap_sys_admin=pe")?.user("hugo"),
```

Comment:
Setting the full "mode" manually would be a massive footgun and as far as I can tell RPM doesn't document that you can do it nor do any of their test specfiles.  They say you can set the "mode" with `%attr()` but actually it seems to only be the permissions.